spec/index.html

```diff
@@ -543,12 +547,20 @@ <h3>RDF Version Announcement</h3>
 	
     <pre class="box http">GET /document.ttl HTTP/1.1
 Host: example.com
-Accept: text/turtle; version=1.2
+Accept: text/turtle; version="1.2"
     </pre>
     <p>
       Section <a href="#defined-version-labels" class="sectionRef"></a> defines <a>version labels</a>
       to be used with the <code>version</code> parameter and in <a>concrete RDF syntax</a>.
     </p>
+
+    <p class="note">As HTTP <a data-cite="webarch#frag-coneg">content negotiation</a> is advisory,
+      clients recieving a document should be prepared to properly handle a document
+      of the requested media type but potentially having a `version` other than what was
+      requested.
+      Systems may consider down-grading the content to an appropriate version
+      as discussed in <a href="#defined-version-labels" class="sectionRef"></a>,
+      or by returning a `406 "Not Acceptable"` status in the response.</p>
```

Comment:
Does "Systems" in the second sentence refer to clients (as mentioned in the first sentence) or to servers?

Reply:
Probably better to use "Clients" to be consistent. I'll do an update for this later today.

Reply:
But clients wouldn't return a 406, as the sentence would suggest after this change.